sdk/src/main/scala/io/horizen/network/SyncStatusActor.scala

```diff
@@ -93,7 +94,7 @@ class SyncStatusActor[
 
   // common method to reset the internal state
   private def resetInternalState(): Unit = {
-    syncStatus = false; processNewBlocks = false
+    syncStatus = false;
```

Comment:
revert all the changes before merging the https://github.com/HorizenOfficial/Sidechains-SDK/pull/778